src/Components/ActionModal.jsx

```diff
@@ -16,11 +16,11 @@ export const ActionModal = ({ setModalVisibility, isModalVisible, video }) => {
     isInPlaylist(playlistToModify, video)

```

Comment:
You can write the `isInPlaylist() on line 35` in a more effective way

```suggestion
 const isInPlaylist = ( playlistName, vidoeUrl ) => playlists.filter( ( listItem ) =>
        listItem.name === playlistName && listItem.itemsInPlaylist.includes( vidoeUrl )
      ).length;
```

This code will still check for both the conditions and return either `0` or `length in positive numbers => 1, 2, etc` and if the conditionals in this line would still work because `0` is treated as `false` and any positive number is treated as `true`